lib/App/MintTag.pm

```diff
@@ -141,7 +141,10 @@ sub ensure_initial_prep ($self) {
     die "local path $dir does not exist! (maybe you should set clone = true)\n"
       unless $self->config->should_clone;
 
-    chdir $dir->parent or die "Couldn't chdir to $dir\'s parent!\n";
+    unless (chdir $dir->parent) {
```

Comment:
This is a weird way to do this; it's possible that chdir could fail because the path exists but is a file, for example.